modules/beanmapping-dozer/pom.xml

```diff
@@ -15,18 +15,49 @@
 
   <dependencies>
-    <dependency>
-      <groupId>com.devonfw.java.modules</groupId>
-      <artifactId>devon4j-basic</artifactId>
-    </dependency>
-	 <dependency>
+	  <dependency>
 	    <groupId>com.devonfw.java.modules</groupId>
       <artifactId>devon4j-beanmapping</artifactId>
     </dependency>
     <dependency>
       <groupId>com.github.dozermapper</groupId>
       <artifactId>dozer-core</artifactId>
     </dependency>
+    <!-- 
+		    [ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.332 s <<< FAILURE! - in com.devonfw.module.beanmapping.common.impl.dozer.BeanMapperImplDozerTest
+				[ERROR] testMapEntity2Eto  Time elapsed: 1.282 s  <<< ERROR!
+				com.github.dozermapper.core.MappingException: 
+				org.xml.sax.SAXException: javax.xml.bind.JAXBException: Implementation of JAXB-API has not been found on module path or classpath.
+				 - with linked exception:
+				[java.lang.ClassNotFoundException: com.sun.xml.internal.bind.v2.ContextFactory]
+				javax.xml.bind.JAXBException: Implementation of JAXB-API has not been found on module path or classpath.
+				 - with linked exception:
+				[java.lang.ClassNotFoundException: com.sun.xml.internal.bind.v2.ContextFactory]
+				  at com.devonfw.module.beanmapping.common.impl.dozer.BeanMapperImplDozerTest.getBeanMapper(BeanMapperImplDozerTest.java:37)
+				  at com.devonfw.module.beanmapping.common.impl.dozer.BeanMapperImplDozerTest.testMapEntity2Eto(BeanMapperImplDozerTest.java:51)
+				Caused by: org.xml.sax.SAXException: 
+				javax.xml.bind.JAXBException: Implementation of JAXB-API has not been found on module path or classpath.
+				 - with linked exception:
+				[java.lang.ClassNotFoundException: com.sun.xml.internal.bind.v2.ContextFactory]
+				  at com.devonfw.module.beanmapping.common.impl.dozer.BeanMapperImplDozerTest.getBeanMapper(BeanMapperImplDozerTest.java:37)
+				  at com.devonfw.module.beanmapping.common.impl.dozer.BeanMapperImplDozerTest.testMapEntity2Eto(BeanMapperImplDozerTest.java:51)
+				Caused by: javax.xml.bind.JAXBException: Implementation of JAXB-API has not been found on module path or classpath.
+				  at com.devonfw.module.beanmapping.common.impl.dozer.BeanMapperImplDozerTest.getBeanMapper(BeanMapperImplDozerTest.java:37)
+				  at com.devonfw.module.beanmapping.common.impl.dozer.BeanMapperImplDozerTest.testMapEntity2Eto(BeanMapperImplDozerTest.java:51)
+				Caused by: java.lang.ClassNotFoundException: com.sun.xml.internal.bind.v2.ContextFactory
+				  at com.devonfw.module.beanmapping.common.impl.dozer.BeanMapperImplDozerTest.getBeanMapper(BeanMapperImplDozerTest.java:37)
+				  at com.devonfw.module.beanmapping.common.impl.dozer.BeanMapperImplDozerTest.testMapEntity2Eto(BeanMapperImplDozerTest.java:51)
+				  
+				  jaxb-api and jaxb-runtime dependencies has been added due to above error
```

Comment:
just wanted to see the trace to understand what is going on. So my guess was correct.
For XML Comment just write something like:
```
<!-- Dozer requires JAX-B what is not included in JDK since version 9+ -->
```

Reply:
Done